### PR TITLE
add action reset volume status for volume admins

### PR DESCRIPTION
**Pull request by andypf**

### CODE REVIEW

**Comment by andypf** [comment]
@Carthaca Please take a look

File: plugins/block_storage/lib/block_storage/driver/block_storage_driver.rb [hunk 1]
```diff
@@ -69,6 +71,23 @@ def test(filter={})
         puts "test"
       end
 
+      def volume_action(id,status={})
```
Comment (Carthaca) [inline plugins/block_storage/lib/block_storage/driver/block_storage_driver.rb]: 'action' is more generic than just for updating the status, maybe rename it to make future extension easier

File: plugins/block_storage/lib/block_storage/driver/block_storage_driver.rb [hunk 2]
```diff
@@ -69,6 +71,23 @@ def test(filter={})
...
+        nedded_keys = ["status","attach_status","migration_status"]
```
Comment (Carthaca) [inline plugins/block_storage/lib/block_storage/driver/block_storage_driver.rb]: nitpick: `needed`

File: app/assets/javascripts/jquery.ajax_paginate.coffee [hunk 3]
```diff
@@ -109,7 +109,6 @@ jQuery.fn.ajaxPaginate= ( options ) ->
         timer = setTimeout(loadAllOnSearch,200)
 
       $(searchInputSelector).keyup (e) ->
-        console.log 'keyup'
```
Comment (Carthaca) [inline app/assets/javascripts/jquery.ajax_paginate.coffee]: debug output?

**Pull request merged**